docs/pages/configuration/api-catalog.md

```diff
@@ -13,7 +13,7 @@ The first step to enable the API Catalog, you need to add a `catalogs` object to
 const config = {
   // ...
   catalogs: {
```

Comment:
I personally think we should always show an array [] and actually, require it. Making the array optional is just making for a dirty conceptual model imo.

Reply:
The scope of this change is only around navigation.